SPECS/kernel/kernel.spec

```diff
@@ -220,6 +223,12 @@ install -vdm 700 %{buildroot}/boot
 install -vdm 755 %{buildroot}%{_defaultdocdir}/linux-%{uname_r}
 install -vdm 755 %{buildroot}%{_prefix}/src/linux-headers-%{uname_r}
 install -vdm 755 %{buildroot}%{_libdir}/debug/lib/modules/%{uname_r}
+
+install -d -m 755 %{buildroot}%{_sysconfdir}/sysconfig
+install -c -m 644 %{SOURCE5} %{buildroot}/%{_sysconfdir}/sysconfig/cpupower
+install -d -m 755 %{buildroot}%{_unitdir}
+install -c -m 644 %{SOURCE6} %{buildroot}%{_unitdir}/cpupower.service
```

Comment:
as @Camelron pointed out you can leave out the `install` lines for the dirs and just use `-D` when installing the files, unless you are specifically creating the dirs separate for a reason?

Reply:
I did consider this initially but not looking to not change the permissions of the directories.